### PR TITLE
Deleting posts

### PR DESCRIPTION
**Pull request by shannongamby**

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -7,13 +7,21 @@ def new
 
   def create
     @post = Post.create(post_params.merge({user_id: current_user.id}))
+    p @post.id
```
Comment (imogenmisso) [inline app/controllers/posts_controller.rb]: can be deleted

**Comment by imogenmisso** [comment]
Good fix! Tests now passing.

**Comment by runticle** [comment]
this is great work todd and shannon!!

**Comment by imogenmisso** [comment]
🌟

**Comment by runticle** [comment]
⭐️

**Comment by rsmith88** [comment]
🌟Great work!

**Comment by runticle** [comment]
woooo! merging! go go go

**Comment by runticle** [comment]
classic merge technique

**Pull request merged**